src/sentry/api/api_owners.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+
+
+class ApiOwner(Enum):
+    """
+    Used to track ownership of APIs
+    Value should map to team's github group
+    """
+
+    DISCOVER_N_DASHBOARDS = "discover-n-dashboards"
+    TELEMETRY_EXPERIENCE = "telemetry-experience"
+    ENTERPRISE = "enterprise"
+    SECURITY = "security"
+    HYBRID_CLOUD = "hybrid-cloud"
+    ISSUES = "issues"
+    PERFORMANCE = "performance"
+    TEAM_STARTFISH = "team-starfish"
```

Comment:
Also why is this one prefixed with `TEAM` but others are not? would that be cause starfish is less of a specific product?

Reply:
I stick with whatever codeowners has and let security-as-code and team owners figure out those details.